Tasks/replace.js

```diff
@@ -1,22 +1,25 @@
-// Replace substring with newstr
+'use strict';
+// Replace substring with newStr
 
-Replace = (str, substr, newstr) => {
+const replace = (str, substr, newStr) => {
   if (substr === '') {
     return str;
-  } else {
-    src = str;
-    res = '';
-    do {
-      const _index = src.indexOf(substr);
-      if (_index === -1) {
-        return res + src;
-      } else {
-        const start = src.substring(0, _index);
-        src = src.substring(_index + substr.length, src.length);
-        res += start + newstr;
-      }
-    } while (true);
   }
+
+  let source = str;
+  let res = '';
+
+  while (true) {
```

Comment:
It's even `replaceAll` :smiley:

Reply:
I know but we don't have to change realization. Also function, that contains only one line it is weird. I thought that here another realization of replaceAll